docs/getting-started/android/setup.md

```diff
@@ -22,6 +22,17 @@
 </tr>
 </table>
 
+### Set up environment variables
+
+fastlane requires some environment variables set up to run correctly. In particular, having your locale not set to a UTF-8 locale will cause issues with building and uploading your build. In your shell profile add the following lines:
```

Comment:
Maybe use `_fastlane_` as in the other places?

Reply:
Good point, thanks 👍